composer.json

```diff
@@ -41,11 +41,6 @@
         "providers": [
             "Italia\\SPIDAuth\\ServiceProvider"
         ]
-    },
-    "patches": {
-      "onelogin/php-saml": {
-        "Compatibility with SPID": "https://rawgit.com/italia/spid-laravel/master/patches/php-saml-3.4.1-spid.patch"
-      }
```

Comment:
@gcastroacn @ste-pastore adesso che è passata la CI, dovreste aprire una nuova PR con la sola aggiunta della patch, in modo che dopo possiamo reintegrare queste linee di codice.

Reply:
Ok, procediamo